software_training_assignment/src/turtlesim_editor/src/turtlesim_move.cpp

```diff
@@ -0,0 +1,67 @@
+#include <ros/ros.h>
+#include <actionlib/server/simple_action_server.h>
+#include <turtlesim_editor/turtlesimAction.h>
+#include <turtlesim/Pose.h>
+#include <geometry_msgs/Twist.h>
+#include <cmath>
+#include "turtle/turtle.h"
+
+class turtlesimAction
+{
+protected:
+  ros::NodeHandle nh_;
+  actionlib::SimpleActionServer<turtlesim_editor::turtlesimAction> as_;
+  ros::Time starttime = ros::Time::now();
```

Comment:
variables should be initialized in the constructor function of a class not in the protected section - smth to keep in mind